### PR TITLE
feature/PingLock

### PR DESCRIPTION
**Pull request by avcio9**

### CODE REVIEW

**Comment by QuentinGruber** [comment]
So we don't just lock there weapons we kick them? I didn't saw it that way but i'm ok with it. @Z1Meme What do you think of it?

File: src/servers/ZoneServer2016/zoneserver.ts [hunk 1]
```diff
@@ -168,6 +168,7 @@ export class ZoneServer2016 extends EventEmitter {
   _db!: Db;
   _soloMode = false;
   _useFairPlay = true;
+  _maxPing = 200;
```
Comment (QuentinGruber) [inline src/servers/ZoneServer2016/zoneserver.ts:171]: should be a public property, servers owner would change this

**Review by QuentinGruber — CHANGES_REQUESTED** [review]

File: src/servers/ZoneServer2016/zoneserver.ts (hunk 1 above)
Comment (avcio9) [reply src/servers/ZoneServer2016/zoneserver.ts]: > So we don't just lock there weapons we kick them? I didn't saw it that way but i'm ok with it. @Z1Meme What do you think of it?

I've talked with @Z1Meme, hes fine with it

**Review by QuentinGruber — APPROVED** [review]

**Comment by Z1Meme** [comment]
> So we don't just lock there weapons we kick them? I didn't saw it that way but i'm ok with it. @Z1Meme What do you think of it?

Looks good, we'll see if we need to raise that ping threshold but it'll be fine for now. We should also add a movement lock for sudden ping spikes using update location.

**Pull request merged**